website/docs/r/compute_instance.html.markdown

```diff
@@ -260,6 +260,16 @@ The `scheduling` block supports:
 * `automatic_restart` - (Optional) Specifies if the instance should be
     restarted if it was terminated by Compute Engine (not a user).
 
+---
+
+* `guest_accelerator` - (Optional, Beta) List of the type and count of accelerator cards attached to the instance. Structure documented below.
```

Comment:
We should link to the Beta section of the provider docs whenever we write that a field is Beta; that way, users can easily discover what that means for them.